crates/bevy_core_pipeline/src/bloom/settings.rs

```diff
@@ -15,8 +15,7 @@ use bevy_render::{extract_component::ExtractComponent, prelude::Camera};
 ///
 /// **Bloom is currently not compatible with WebGL2.**
 ///
-/// Often used in conjunction with `bevy_pbr::StandardMaterial::emissive`
-/// for 3d meshes or [`bevy_render::color::Color::Hsla`]`::lightness` for 2d sprites.
+/// Often used in conjunction with `bevy_pbr::StandardMaterial::emissive` for 3d meshes.
```

Comment:
I suppose for sprites / mesh 2d one can use the color parameter. Just that that is assumed to be non-linear sRGB.